Assets/Scripts/UI/Pickup Pup/RedeemPanel/RedeemDisplay.cs

```diff
@@ -0,0 +1,73 @@
+﻿/*
+ * Author: James Hostetler
+ * Description: Controls the Redeem Display.
+ */
+
+using UnityEngine;
+using UnityEngine.UI;
+using System.Collections;
+
+public class RedeemDisplay : PPUIElement {
+  
+    [SerializeField]
+	Text giftDescription;
+	PPLivingroomUIController room;
+	GiftItem gift;
+	Image background;
+	CanvasRenderer canv;
+
+	// Need Eventually For Polish
+	[SerializeField]
+	Button RedeemButton;
+	[SerializeField]
+	Button RedeemReturnButton;
+
+
+	protected override void setReferences()
+	{
+		base.setReferences();
+		background = GetComponent<Image>();
+		canv = GetComponent<CanvasRenderer>();
+	}
+
+	protected override void fetchReferences()
```

Comment:
No need to override functions if you're not adding any additional calls beyond the base's functionality